Build fails out-of-the-box for VS 2019 (16.6.5)

Compilation fails in Balance.cpp:

* error C3849: function-style call on an expression of type 'const BalanceListenerSort' would lose const and/or volatile qualifiers for all 3 available operator overloads (compiling source file ..\util\Balance.cpp)

I assume that this is because the BalanceListenerSort comparison operators are not explicitly const.

--- a/main/source/util/Balance.cpp
+++ b/main/source/util/Balance.cpp
@@ -33,11 +33,11 @@ using std::vector;
 
 struct BalanceListenerSort
 {
-	bool operator()(const BalanceChangeListener* lhs, const BalanceChangeListener* rhs)
+	bool operator()(const BalanceChangeListener* lhs, const BalanceChangeListener* rhs) const
 	{ return lhs->getBalanceChangeListenerID() < rhs->getBalanceChangeListenerID(); }
-	bool operator()(const BalanceChangeListener* lhs, const unsigned int rhs)
+	bool operator()(const BalanceChangeListener* lhs, const unsigned int rhs) const
 	{ return lhs->getBalanceChangeListenerID() < rhs; }
-	bool operator()(const unsigned int lhs, const BalanceChangeListener* rhs)
+	bool operator()(const unsigned int lhs, const BalanceChangeListener* rhs) const
 	{ return lhs < rhs->getBalanceChangeListenerID(); }
 };
 

